part-07.mdx

```diff
@@ -0,0 +1,126 @@
+
+You can also enable data padding and Bézier lines on charts.
+
+### Locale
+
+This tab houses options to scale and select units for time and temperature.
+
+You can also choose either to use browser local time or server time. 
+
+However, note that these settings are applied gradually as charts are updated. 
+To force them, refresh the dashboard now. 
+
+![dash-settings](https://user-images.githubusercontent.com/12263278/64901553-967f3880-d692-11e9-95ac-dd485d36535c.gif)
+
+## Check if there's an update to Netdata
+
+Netdata upon installation with [the one-liner](https://docs.netdata.cloud/packaging/installer/#one-line-installation) 
+comes with the auto-update option enabled by default.
+
+You can always check if there is an update available right from the Update tab of your Netdata dashboard.
+
+![nedata-update](https://user-images.githubusercontent.com/12263278/64876743-be957a00-d647-11e9-83dd-2f0a8df572cb.gif)
+
+You can see an update is available from the above short screen record.
+
+Netdata automatically updates itself, but if you choose to update it manually, there are [several well-documented methods](https://docs.netdata.cloud/packaging/installer/update/) to achieve that. 
+However, it is best practice for you to first go over the [changelog](https://docs.netdata.cloud/changelog/).
+
+## Export or import a snapshot
+
+Snapshot files include all the information of the dashboard, including the URL of the origin server, 
+its netdata unique ID, and chart data queries for the visible timeframe,
+so align, pan and zoom the charts as needed. 
+Snapshots are not in real-time. 
+
+Netdata can export and import dashboard snapshots. Any Netdata can import the snapshot of any other Netdata. 
+
+You download the snapshot to your computer as a file, 
+that can be imported back into any Netdata dashboard, 
+you do not need to import it back to the current server.
+
+Snapshots don't upload to a server; instead, the web browser on your computer handles them.
+
+They are temporary because, on the reload of your browser, the actual Netdata dashboard replaces the snapshot.
+
+### To export or save a snapshot
+
+ Click on the top upload icon as seen on the short clip below, edit the snapshot file name and 
+ finally select your desired compression method and click `Export`.
+
+ ![export-shot](https://user-images.githubusercontent.com/12263278/64901454-48b60080-d691-11e9-9c14-1539bc841735.gif)
+
+ You save the snapshot file after it gets generated on your browser and like a regular file attachment, 
+ you can email it to your colleague.
+
+ Congratulations, you have successfully exported a snapshot.
+
+ #### To import or load a snapshot
+
+ Click on the top download icon as seen on the short clip below. 
+
+![import-shot](https://user-images.githubusercontent.com/12263278/64901503-ee696f80-d691-11e9-9678-8d0e2a162402.gif)
+
+ Select the Netdata snapshot file to import. 
+
+ Once the file is loaded, click `import` to render it.
```

Comment:
This is the point where I'd explain that refreshing Netdata will then erase the snapshot data and bring you back to real-time updates.